### PR TITLE
Bump rust nigthly to build sill

### PR DESCRIPTION
**Pull request by Etn40ff**

### CODE REVIEW

File: rust/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ARG FROM
 FROM $FROM
 
-RUN echo "bumping on 2022/01/22"
+RUN echo "bumping on 2022/10/29"
```
Comment (LinusCDE) [inline rust/Dockerfile]:
```suggestion
RUN echo "bumping on 2023/03/08"
```

My fault ig guess for leaving overlooking this. While at it, it should be the most recent one to date ig.

**Review by LinusCDE — APPROVED** [review]
Sorry for overlooking this. Just got to update plato after a while and also had an issue building due to outdated rust (https://github.com/toltec-dev/toltec/pull/669).

Added a suggestion to update to the current date. In future we should probably pin nightlies to a specific date which is possible, but not guaranteed a given date has all the required components.

For now this should suffice. In order to publish the newest plato, I'll also need this merged. @matteodelabre mind checking/approving this and publishing a new version with this included (e.g. tag v2.3.1)?

**Comment by LinusCDE** [comment]
Also not sure if the build error is just because it's a PR or some kind of deprecation regarding the use of `docker login`. I suppose it's the first but just wanted to note it.

https://github.com/Etn40ff/toolchain/actions/runs/4367584204/jobs/7639086007:
```
Error: Cannot perform an interactive login from a non TTY device
```

**Comment by Eeems** [comment]
I just opened up a new PR and it's not encountering this error. It might be fixed? Could you try pushing a new empty commit or something similar to trigger a new build?

**Comment by Eeems** [comment]
Huh, that's odd. My PR built fine. Maybe it's something to do with my PR being a branch on this repo and not a fork.

**Comment by Eeems** [comment]
I've opened #19 which worked. I'll close this and merge that.

**Pull request closed without merging**